### PR TITLE
Bug 1961120: added permissions to service monitoring

### PR DESCRIPTION
**Pull request by Elbehery**
This PR added permissions to operators service monitoring, allowing cluster version upgrade.

Currently the operators has `create` permission. When upgrading the cluster, it needs permission to `update` the service role.

### CODE REVIEW

File: manifests/10_deployment-ibm-cloud-managed.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# *** AUTOMATICALLY GENERATED FILE - DO NOT EDIT ***
```
Comment (jsafrane) [inline manifests/10_deployment-ibm-cloud-managed.yaml:1]: Why did you remove this comment?

Comment (Elbehery) [reply manifests/10_deployment-ibm-cloud-managed.yaml:1]: I did not, I run `make update` after I changed the `yamls` 

btw, does the `yaml` need a new line at the end ?

Comment (jsafrane) [reply manifests/10_deployment-ibm-cloud-managed.yaml:1]: `make verify` does not like this change.

Comment (Elbehery) [reply manifests/10_deployment-ibm-cloud-managed.yaml]: Yes, this line was removed after executing `make update`. 
So I have `manually` restored the file, after executing `make update`, `make`, and `make test`
